### ISSUE TITLE
NMM 0.62.14 - Warning for Multi-HD Virtual Install path in Virtual folder settings

### ISSUE THREAD

**Issue by KunoMochi** [Nexus-Mods/NMM-Test-Builds#97]
**Issue:**
A warning icon pops up when you go into the _Change Virtual folders..._ settings and try to just change the Virtual Install path.

Pressing _Cancel_, closing the window, or flipping the Multi-HD option off then back on before hitting _Finish_ are the only ways to get out.

**Steps to reproduce:**
1. Launch NMM and be using Multi-HD mode
2. Go to _Tools_ > _Change Virtual folders..._
3. Change only the Virtual Install path (or don't) and leave Multi-HD on
4. Click on _Finish_

You should then see a flashing red Warning icon and not letting you proceed.

![nmm0-62-14_multihdpathwarning](https://cloud.githubusercontent.com/assets/7691651/15391619/1a6cc38e-1d90-11e6-877c-9026ff73e21e.png)

**Comment by DuskDweller** [comment]
Can you mouse hover the red icon to check what's the issue?

**Comment by KunoMochi** [comment]
**Update:**
Warning icon states the following message:

```
You MUST set the Virtual Directory on the same HD as the usual mod install folder:
Selected HD: E:\ - Expected HD: d:\
```
- Game is installed on D: drive
- Virtual Directory is on E: drive

![nmm0-62-14_multihdpathwarning2](https://cloud.githubusercontent.com/assets/7691651/15410186/72ca898e-1de6-11e6-9379-e544296f63ca.png)

**Additional note:**
With Multi-HD mode off, the first text field requires the path to be on the same drive as the game or mod installs.

With Multi-HD mode on, the path on the first field is now moved to the second text field and the first text field is needed to be changed.

I was wondering if there would be a way to leave the first field as is and only need to change the second field when you enable Multi-HD mode.

**Comment by DontBlnkBadWolf** [comment]
Yeah, I myself have experienced this as well. As my OCD dictates for me. I would like to have every folder to be in one area, instead of the current way they want to be placed.

**Comment by DuskDweller** [comment]
With the current install method if you're using multihd mode, there's no way around it, cause some files can't be physically located on a different drive than the game's.

Anyway, after this profile sharing is finally released there are a few improvements we'll be adding related to this, mainly a new install method that won't create links around and allows you to set your install folder wherever you wish.

**Comment by DontBlnkBadWolf** [comment]
Awesome, now maybe I can have all the folders in the C:\Program Files (x86)\Steam\SteamApps\common\Fallout 4 folder instead of being everywhere. My OCD Thanks you

**Comment by NikaAlexandrov** [comment]
I have a question about this? What about those of us who don't want the freaking stupid thing to be on stupid freaking c drive? I certainly don't? I want it on a drive that has more space, not less as c drive on my pc certainly does.

**Comment by jonSP12** [comment]
it is just so stupid, that its impossible for the human brain to even compreend such a thing,
This thing updates automatically and someone is trolling with it 

already changed all my directories to  d:\ , c:\ 

![kjfdghkdfjhg](https://github.com/Nexus-Mods/NMM-Test-Builds/assets/77213594/3a883ee8-b783-4167-9da0-e9f58920e952)
